src/pages/ImportData.js

```diff
@@ -124,8 +124,24 @@ const ImportDataOverview = ({ header, snapshots, isDataset, snapshotResponses, u
         ])
       : url && div({ style: { fontSize: 16 } }, ['From: ', new URL(url).hostname]),
     div({ style: { marginTop: '1rem' } }, [
-      `The ${isDataset ? 'dataset' : 'snapshot'}(s) you just chose to import to Terra will be made available to you `,
-      'within a workspace of your choice where you can then perform analysis.',
+      Utils.cond(
+        [
+          isProtected(url, format),
+          () =>
+            '⚠️  The data you chose to import to Terra are identified as protected and require additional security settings. Please select a workspace that has an Authorization Domain and/or protected data setting.',
```

Comment:
We use a warning icon elsewhere (`icon('warning-standard')`) instead of a unicode character.

Reply:
Done!